openforcefield/utils/toolkits.py

```diff
@@ -1159,7 +1159,11 @@ def generate_conformers(self, molecule, n_conformers=1, clear_existing=True):
         status = omega(oemol)
 
         if status is False:
-            raise Exception("OpenEye Omega conformer generation failed")
+            omega.SetStrictStereo(False)
```

Comment:
Do we want to at least issue a warning that we're trying to parameterize a molecule with unspecified stereochemistry, meaning we may get incorrect charges?

Reply:
This took some thought -- Per your review message
> However, it might be fine that the toolkit just gives its "best effort" here silently, since we've already had to force this when we created the Molecule.

This was my perspective as well -- If the user already forced `allow_undefined_stereo=True` earlier, then they know what they're getting into. With the current toolkit behavior, `allow_undefined_stereo` has established a secure perimeter -- Beyond using the API to make a stereo-ambiguous molecule atom-by-atom, there shouldn't be any way to get a molecule missing stereochemistry to this point without saying `allow_undefined_stereo=True` at least once. Making users specify it again (in `create_openmm_system`) would be a nuisance that doesn't add much.